java-source/src/test/java/com/example/contract/IOUFlowTests.java

```diff
@@ -52,8 +52,7 @@ public void flowReturnsASignedTransaction() throws InterruptedException, Executi
         ListenableFuture<SignedTransaction> future = a.getServices().startFlow(flow).getResultFuture();
         net.runNetwork(-1);
 
-        Object signedTx = future.get();
-        assert(signedTx instanceof SignedTransaction);
+        SignedTransaction signedTx = future.get();
```

Comment:
Don't need to assign it to a variable that's not used. Compiler will warn about it.